### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by shaurya415** [community/community#63907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to get achievements faster?

**Comment by ghost** [comment]
Consider reading this https://github.com/4xmen/Get-Github-Achievements

> have a nice day ahead.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
